src/windows-emulator/minidump_loader.hpp

```diff
@@ -0,0 +1,17 @@
+#pragma once
+#include <filesystem>
+
+class windows_emulator;
+
+class minidump_loader
+{
+  public:
+    minidump_loader(windows_emulator& win_emu, std::filesystem::path minidump_path);
+    ~minidump_loader();
+
+    void load_into_emulator();
+
+  private:
+    windows_emulator& win_emu_;
+    std::filesystem::path minidump_path_;
+};
```

Comment:
feels like this class could just be a function: `void load_minidump_into_emulator(windows_emulator& win_emu, std::filesystem::path minidump_path)`